### ISSUE TITLE
I just heard about this library and

### ISSUE THREAD

**Issue by santosmarco-caribou** [arielhs/ts-arithmetic#2]
Great work! Awesome job! I like it very much. 

Thank you for this.

**Comment by homomorphist** [comment]
Indeed, I just found it right now, and it's spectacular! I was actually trying to do a bit of research to attempt to make something similar (as a personal project, not to be released) when I ended up stumbling upon this. 

My attempt can take longer than 10 seconds to do what yours can do nearly instantly.  The only one up I've got is that it can go beyond the integer limit since it uses strings instead, but that's not really too useful.

Fantastic work; I'll definitely be taking a peek to see how all of this was done. I've clearly got a ways to go, haha. I mean, jeez, you've even got floats. Anyways, sorry for the ramble, I'm way too tired to function properly. Great work!

**Comment by arielhs** [comment]
Thank you for the kind words! Maybe I'll add another .md file explaining the techniques used to achieve what i have here. I learnt a lot through this process and it would be good to share. I'll remember to tag you if i ever do

**Comment by unao** [comment]
Extremely impressive indeed. This demonstrates your TS mastery. 🧙

**Comment by erhant** [comment]
I have also just stumbled upon this, blew my mind. I tried to do basic arithmetic using bitwise logic and stuff but always ran into recursion depth issues at some point; this lad here does not only that but floating point too 🤯 

pls show us the way 🙏🏻 🙏🏻
